### PR TITLE
realtime-follow-notification-1

### PR DESCRIPTION
**Pull request by longnv-99**
## Related Tickets:
- [#42146] https://edu-redmine.sun-asterisk.vn/issues/42146

## WHAT this PR does?
- Realtime notification when being followed

## Notes Impacted Areas:
- N/A.

## Attachments:
- 

https://user-images.githubusercontent.com/86346191/135709570-39eebff0-6e13-4efb-925a-d695c0515b7a.mp4

### CODE REVIEW

**Review by truongcr7 — APPROVED** [review]

**Comment by longnv-99** [comment]
ready

File: app/Notifications/FollowNotification.php [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+     * Get the mail representation of the notification.
+     *
+     * @param  mixed  $notifiable
+     * @return \Illuminate\Notifications\Messages\MailMessage
+     */
+    public function toMail($notifiable)
+    {
+        return (new MailMessage)
+            ->line('The introduction to the notification.')
+            ->action('Notification Action', url('/'))
```
Comment (manhtd-1083) [inline app/Notifications/FollowNotification.php:50]: sao message chung chung thế :\

Comment (longnv-99) [reply app/Notifications/FollowNotification.php:50]: @manhtd-1083 cái này ko dùng đến b ơi

**Comment by longnv-99** [comment]
ready

File: app/Notifications/FollowNotification.php (hunk 1 above)
Comment (manhtd-1083) [reply app/Notifications/FollowNotification.php]: ko dùng xóa đi chứ :\

**Pull request merged**